src/routes/SubmissionRoutes.js

```diff
@@ -8,33 +8,38 @@ module.exports = {
       controller: 'SubmissionController',
       method: 'createSubmission',
       auth: 'jwt',
-      access: ['Topcoder User', 'Administrator', 'Copilot']
+      access: ['Topcoder User', 'Administrator', 'Copilot'],
+      scopes: ['write:submission', 'all:submission']
     }
   },
   '/submissions/:submissionId': {
     get: {
       controller: 'SubmissionController',
       method: 'getSubmission',
       auth: 'jwt',
-      access: ['Topcoder User', 'Administrator', 'Copilot']
+      access: ['Topcoder User', 'Administrator', 'Copilot'],
+      scopes: ['read:submission', 'all:submission']
     },
     put: {
       controller: 'SubmissionController',
       method: 'updateSubmission',
       auth: 'jwt',
-      access: ['Administrator']
+      access: ['Administrator'],
+      scopes: ['write:submission', 'all:submission']
     },
     patch: {
       controller: 'SubmissionController',
       method: 'patchSubmission',
       auth: 'jwt',
-      access: ['Administrator']
+      access: ['Administrator'],
+      scopes: ['write:submission', 'all:submission']
     },
     delete: {
       controller: 'SubmissionController',
       method: 'deleteSubmission',
       auth: 'jwt',
-      access: ['Administrator']
+      access: ['Administrator'],
+      scopes: ['write:submission', 'all:submission']
```

Comment:
same issue as PATCH comment above. Do we even allow members to retract/delete their submission @ajefts ?

Reply:
@cwdcwd we do in some cases.  One example is for design challenges, you have the concept of managing your submissions so you can add/remove while submission phases are active